### ISSUE TITLE
Table layout shows images squished on npm

### ISSUE THREAD

**Issue by kentcdodds** [all-contributors/all-contributors-cli#161]
For example: https://www.npmjs.com/package/all-contributors-cli

![image](https://user-images.githubusercontent.com/1500684/52365307-70aafa80-2a04-11e9-8fa8-f14a6da020cc.png)

Would there be a way to make sure the aspect ratio remains the same?

**Comment by Berkmann18** [comment]
There are ways to do that but I'm not sure if using things like `max-width: ...; max-height: ..., width: auto; height: auto` or `object-fit: contain` would help considering that's it's tricky to get the CSS right in GH Markdown.
I haven't tried any approaches in this context (yet) but will have a go (unless someone else goes for it) tomorrow when I'll get the time.

**Comment by jakebolam** [comment]
For refrence: (previous table)
https://www.npmjs.com/package/all-contributors-cli/v/5.11.0
<img width="882" alt="screen shot 2019-02-06 at 9 11 10 am" src="https://user-images.githubusercontent.com/3534236/52367035-33883d80-29ef-11e9-8854-a39e44854565.png">

**Comment by jakebolam** [comment]
Contents inside cells are not centered anymore either

**Comment by jakebolam** [comment]
Thoughts on a rollback? I feel that given the major version increment, we should be safe to roll-forward with a fix.

For future reference, we may want to consider 'beta' releases for big changes.

**Comment by Berkmann18** [comment]
@jakebolam I'm okay with a rollback and for _beta_ releases.

**Comment by kentcdodds** [comment]
I don't think we need to rollback for this personally.

**Comment by jakebolam** [comment]
This is due to the height tag we added. I'm going to remove this change. 

TLDR: npm isn't respecting the width attribute, it shrinks the width. But the height is still respected.

**Comment by kentcdodds** [comment]
Weeeird!

**Comment by jakebolam** [comment]
@Berkmann18 had some time this afternoon and got a fix up: https://github.com/all-contributors/all-contributors-cli/pull/164/files
